'leg_joints':
- r_hip_yaw
- r_hip_roll
- r_hip_pitch
- r_knee
- r_ankle_pitch
- r_ankle_roll

'angle_stop':
- 0
- 0
- 0
- 0
- 0
- 0

'angle_e_stop':
- 0
- 1
- 0
- 0
- 0
- 0

'angle_forward':
- 1
- 0
- 0
- 0
- 0
- 0

